add-Olabanji121

```diff
@@ -0,0 +1,6 @@
+Refactoring Steps
```

Comment:
Make this heading bold
`**Refactoring Steps**`

Reply:
I have made the changes